src/main/java/megamind/ui/Ui.java

```diff
@@ -0,0 +1,79 @@
+package megamind.ui;
+
+/**
+ * The `Ui` class handles interactions with the user.
+ * It provides methods to display messages, errors, and task-related information.
+ * The class also includes methods to greet the user and show help information.
+ */
+public class Ui {
+    private static final String HORIZONTAL_LINE = "____________________________________________________________";
+
+    /**
+     * Greets the user.
+     */
+    public String greet() {
+        String logo = """
```

Comment:
I like how you changed the logo of the original chatbot to make it look more like megamind!